### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD014.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#20] [General]
# https://blog.3place.cn/3place/fsd/FSD014.html

https://blog.3place.cn/3place/fsd/FSD014.html

**Comment by picklecai** [comment]
梅非斯特
我们只要写张有效的证明
声称她的死去的夫
葬在帕多瓦神圣的墓地

浮士德
好主意！我们首先要去作一次旅行

梅非斯特
神圣的单纯！此事大可不必
证明毋需知道详情

浮士德
你若别无良谋，此计无法执行

梅菲斯特：不知而作之。
浮士德：我无是也。
